sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -598,6 +598,16 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createOptional
 
+  val ADAPTIVE_SHUFFLE_HASH_JOIN_LOCAL_MAP_THRESHOLD =
+    buildConf("spark.sql.adaptive.shuffledHashJoinLocalMapThreshold")
+      .doc("Configures the maximum size in bytes for per partition that can be allowed to build " +
+        "local hash map. If all the partition size not larger than this threshold, join " +
+        s"selection may use shuffled hash join instead of sort merge join when " +
+        s"${PREFER_SORTMERGEJOIN.key} is false.")
+      .version("3.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("64MB")
```

Comment:
curious why we choose this default value? to be same as `spark.sql.adaptive.shuffle.targetPostShuffleInputSize`?

Reply:
The main idea is that the default skew join size is `256MB` and the local map should smaller 3x(follow the existed formula) than other side. So assume the local map size is `64MB` and other side is `192MB`.